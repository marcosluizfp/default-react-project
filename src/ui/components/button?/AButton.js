import React from 'react';

export default function AButton() {
  return <div>Hello World Button!</div>;
}
