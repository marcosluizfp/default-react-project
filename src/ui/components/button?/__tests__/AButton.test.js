/* eslint-disable no-undef */
import { shallow } from 'enzyme';
import React from 'react';
import AButton from '../AButton';

describe('<AButton />', () => {
  it('expect to match snapshot', () => {
    expect(shallow(<AButton />)).toMatchSnapshot();
  });
});
