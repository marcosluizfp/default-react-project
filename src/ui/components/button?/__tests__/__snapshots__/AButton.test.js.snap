// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AButton /> expect to match snapshot 1`] = `
<div>
  Hello World Button!
</div>
`;
